README.md

```diff
@@ -9,27 +9,82 @@ Note that at this time the Package Manager has no support for iOS, watchOS, or t
 ---
 
 ## Table of Contents
-
-* [Contributions](#contributions)
+* [Getting Started](#getting-started)
+* [Documentation](#documentation)
 * [System Requirements](#system-requirements)
 * [Installation](#installation)
   * [Managing Swift Environments](#managing-swift-environments)
   * [Choosing a Swift Version](#choosing-a-swift-version)
-* [Documentation](#documentation)
 * [Support](#support)
+* [Contributions](#contributions)
 * [License](#license)
 
 ---
 
-## Contributions
+## Getting Started
+🧐 Verify Installation 
+```
+$  swift package --version
+```
+🏁 Initialize Package
```

Comment:
I think its better if we just put a link to https://swift.org/getting-started/#using-the-package-manager. The advantage is, we don't have to maintain similar documents in two places. However, unfortunately, the Swift.org website is not open source so people can't directly contribute to it. But, you can open a JIRA issues with suggestions and I can update the blog!

Reply:
One thing that is greatly missing from Swift.org’s getting started is adding dependencies though. And that was Michael’s big pain point. How do you suggest we improve that?

Reply:
I think, for now, we can enhance Swift.org posts using JIRA. If you open a JIRA issue with the text you want me to add/change on the website, I can do that! I am sorry it has to be awkward like this. I'll email Ted about this.

Reply:
Hey @aciidb0mb3r , i agree it will be easier to maintain. We can display a example of using swiftpm in --help command ? And should i remove the getting started section or will it be good to reduce it to  2-3 commands and a sample Package.swift

Reply:
Since I used the package manager for the first time I also feel this is an Extraordinary guide! https://swift.org/getting-started/#using-the-package-manager

@hartbit for the dependencies I think there should be a next tutorial like the way Django do a tutorial series like this : https://docs.djangoproject.com/en/2.0/intro/tutorial01/
Why I think this approach is better as for first time users or Students in general, Having everything at a same place can be too much data as people get worried when they see so much information thrown like that. Just my point of view though :)

Reply:
just an idea we can add liltle documentations of common commands on init command ... so maybe something like this displayed after initializing a package ..
<img width="610" alt="screen shot 2018-02-20 at 1 02 12 pm" src="https://user-images.githubusercontent.com/22680912/36412053-7f8b499a-163e-11e8-8c8a-958b14bf38d0.png">

Reply:
Its already there I guess. Are you suggesting UI changes? 
<img width="819" alt="screen shot 2018-02-20 at 1 09 18 pm" src="https://user-images.githubusercontent.com/5003400/36412225-50503b62-163f-11e8-8738-b20813d06e92.png">

Reply:
it shows all commands we can add most common ones with ```swift package init``` and maybe an example of using this commands in help then it will be able to give help of specific commands like for example if there are flags available with init commands then ``` swift package help init ``` will show all possible flags of build command like one for creating library and one for executable

Reply:
umm I agree to you but I guess we have to keep in mind of both beginner users and advanced users. Generally advanced developers work on multiple languages and they look at help to quickly get the commands and then do the work. Maybe the above solution can make things worse for normal and advanced users. I am not sure if we can do this Swift package --help --first or Swift package --help --beginner and your solution for Swift package --help --init sounds good to me :)

Reply:
Actually, i found it is present their already ```swift package init --help```

Reply:
We definitely need to improve the command line user experience in a way that it is delightful to use for both beginner and advanced users. But that is going to be a major change and should probably happen via a swift-evolution proposal. You should feel free to propose such changes if you have a specific proposal in mind!

Reply:
Awesome! I will give it a shot :D

Reply:
For this particular PR, I suggest we just do something simple like this:

```
## Getting Started

Please follow the following link for learning how to use the package manager basics:
https://swift.org/getting-started/#using-the-package-manager
```

Reply:
@aciidb0mb3r  should i close this for now?